apps/trackers/tests/test_jira.py

```diff
@@ -2367,7 +2373,11 @@ def test_generate_query_multiflaw(self):
             source="REDHAT",
             cwe_id="CWE-1",
         )
-        flwcvss = FlawCVSSFactory(flaw=flaw, issuer=FlawCVSS.CVSSIssuer.REDHAT)
+        flwcvss = FlawCVSSFactory(
```

Comment:
I guess that this was supposed to be `flawcvss` instead like it is above

Reply:
This is intentionally, see `flwcvss2`, `flwcvss3`...